scripts/persistent-docker-setup/conf/rusk.toml

```diff
@@ -0,0 +1,56 @@
+# log_type = 'coloured' (default)
```

Comment:
If we want to have a sample rusk.toml file we should have it along with the sample genesis.toml, @herr-seppia should be able to tell a better location for this

Reply:
It's not a sample. It's the default.

Reply:
I'm not sure about having a duplicate one, can we have the docker image curl the config file from the url?

Reply:
Okay, done.